### PR TITLE
Change get_region and get_region_info to use a common implementation

### PR DESCRIPTION
**Pull request by sunxiaoguang**
## What have you changed? (mandatory)
There are some code in get_region and get_region_info can share a common implementation instead. 

## What are the type of the changes? (mandatory)
Improvement

## How has this PR been tested? (mandatory)
Manually against PD server

## Does this PR affect documentation (docs/docs-cn) update? (mandatory)
No

## Does this PR affect tidb-ansible update? (mandatory)
No

### CODE REVIEW

File: src/pd/client.rs [hunk 1]
```diff
@@ -63,6 +63,38 @@ impl RpcClient {
     pub fn get_leader(&self) -> Member {
         self.leader_client.get_leader()
     }
+
+    #[inline]
+    fn option() -> CallOption {
```
Comment (siddontang) [inline src/pd/client.rs]: call_option may be better?

**Comment by siddontang** [comment]
Thanks @sunxiaoguang 

Please sign off your all commits. See https://github.com/tikv/tikv/blob/master/CONTRIBUTING.md#signing-off-the-commit

If you have already pushed some commits before, you can use `git rebase -i master --signoff` to resign them off.

File: src/pd/client.rs (hunk 1 above)
Comment (sunxiaoguang) [reply src/pd/client.rs]: Sure, this eliminates ambiguity. Let me change it.

**Pull request merged**